src/GovUk.Education.ExploreEducationStatistics.Content.Services.Tests/ThemeServiceTests.cs

```diff
@@ -1818,7 +1984,7 @@ await using (var context = InMemoryContentDbContext(contextId))
         }
 
         [Fact]
-        public async Task GetPublicationTree_DataTables_DoesFilterPublicationWithoutDataOnLatestRelease()
+        public async Task GetPublicationTree_DataTables_DoesNotFilterPublicationWithDataOnPreviousRelease()
```

Comment:
Just to be consistent with some of these other test cases (and for brevity), maybe we could call this `GetPublicationTree_DataTables_PreviousReleaseHasData`